src/test/test.js

```diff
@@ -40,12 +45,12 @@ let expectTreeEqual = function(tree1, tree2, ignoreEmptyFolders, checkOrder = tr
         : tree2.children
       expect(children1).to.have.length(children2.length)
       children2.forEach((child2, i) => {
-        expectTreeEqual(children1[i], child2, ignoreEmptyFolders, checkOrder)
+        expectTreeEqualRec(children1[i], child2, recDepth + 1, ignoreEmptyFolders, checkOrder)
       })
     }
   } catch (e) {
     console.log(
-      `Trees are not equal: (checkOrder: ${checkOrder}, ignoreEmptyFolders: ${ignoreEmptyFolders})`,
+      `Trees are not equal: (recDepth: ${recDepth}, checkOrder: ${checkOrder}, ignoreEmptyFolders: ${ignoreEmptyFolders})\n`,
```

Comment:
This is to improve readability? Cool :)

Reply:
Yes. When the mismatch is in a subtree, the print only shows the mismatching subtrees. This makes is easier to find those subsections within the overall trees being compared. I found it helpful while debugging.